woocommerce-gateway-ebanx/gateways/class-wc-ebanx-gateway.php

```diff
@@ -970,6 +970,15 @@ protected function process_response($request, $order)
 				update_post_meta($order->id, "_ebanx_payment_refunds", $refunds);
 				break;
 			case 'UPDATE':
+				$status = array(
+					'CO' => 'Confirmed',
+					'CA' => 'Canceled',
+					'PE' => 'Pending',
+					'OP' => 'Opened'	
+				);
+
+				$order->add_order_note(sprintf(__('EBANX: The payment has been updated to: %s.', 'woocommerce-gateway-ebanx'), $status[$data->payment->status]));
```

Comment:
Won't it be confusing? The merchant will get a order status and a payment status at the same time. And sometimes it is not needed, like when the payment was confirmed, woocommerce already tells the merchant it.